### DISCUSSION TITLE
Tailwind CSS presets not working in a Nuxt project on Windows 11

### DISCUSSION THREAD

**Discussion by llops** [primefaces/community#918] [PrimeVue]
Hi!
Following the steps in this video [Setting Up PrimeVue, Tailwind CSS in a Nuxt Project](https://www.youtube.com/watch?v=q5ck18NwYz8&t=9s), I have not been able to make it work on Windows 11.
As someone points out in the comments, it gives an error related to this line:
`importPT: { from: path.resolve(__dirname, './presets/lara/') }`

I have directly downloaded the example from the repository ([primevue-examples/nuxt-unstyled-tailwind/](https://github.com/primefaces/primevue-examples/tree/main/nuxt-unstyled-tailwind)) and indeed, it fails in the imporPT:
` ERROR  Failed to resolve import "D:GitProjectsuxt-unstyled-tailwindpresetslara" from "virtual:nuxt:D:\GitProjects\nuxt-unstyled-tailwind\.nuxt\primevue-plugin.mjs". Does the file exist?`

Can you test that example on a Windows machine, please?

Thanks for your help!

**Comment by seba-1911** [comment]
Hi,
"as:" is still missing. 

`importPT: { from: path.resolve(__dirname, './presets/lara/'), as: 'tailwind' }`

**Comment by Q-Mick** [comment]
I had the same issue. I just changed it to this.
importPT: { from: './presets/lara/' }

**Comment by llops** [comment]
Thank you very much for your help, @seba-IP @Q-Mick.

In the end, what worked for me is:
`importPT: { as: 'Lara', from: '@/presets/lara/' },`
I found the solution at the end of [this thread](https://github.com/primefaces/primevue-nuxt-module/issues/6)

It would be great if the team could update the documentation and examples with Windows users in mind, because many people are going to encounter this problem...

Thank you!
